contracts/contracts/gho/gho-core/src/contracts/gho/ERC20.sol

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: MIT-only
+pragma solidity ^0.8.0;
+
+import {IERC20} from "openzeppelin-contracts-gho/token/ERC20/IERC20.sol";
```

Comment:
This location isn't found. Could you ensure that you are importing from the right source. Ideally, yo want to get the contract to compile using `npx hardhat compile`.

Reply:
had you done an ```npm install``` ?

Reply:
No, I hadn't. It compiles now. Thanks